tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Process/ProcessProxy.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="ProcessProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Process;
+
+/// <summary>
+/// System.Diagnostics.ProcessProxy struct proxy
+/// </summary>
+[DuckCopy]
+internal struct ProcessProxy
```

Comment:
Do you use this anywhere? I would think you can access `System.Diagnostics.Process` directly in any runtime without a proxy.